docs/setup_vscode.md

```diff
@@ -32,7 +32,7 @@ $ rm -rf p1-stats              # Delete
 ```
 
 ## Install
-Choose your platform below.
+Choose your platform below. Also make sure to install extensions.
```

Comment:
I added this since now the install of the extensions happens in a separate section than the platform-specific install. That means it might be easier for a student to finish the "macOS" section, but not realize they also need "Extensions > macOS".